rules-realm/boostsecurityio/stored-secrets/rules.yaml

```diff
@@ -1,11 +1,3492 @@
 rules:
   CWE-200:
     name: CWE-200
-    pretty_name: CWE-200 - Exposure of Sensitive Information to an Unauthorized Actor
+    pretty_name: Exposed Secret - CWE-200
     group: stored-secrets
     ref: https://cwe.mitre.org/data/definitions/200.html
-    description: The repository exposes sensitive information to an actor that is
-      not explicitly authorized to have access to that information.
+    description: The repository exposes sensitive information (Cwe 200)
+      to an actor that is not explicitly authorized to have access to that information.
```

Comment:
Not sure if we should change the pretty name & description of this rule, it's currently in use by gitleaks.

Reply:
I wasn't sure about that. There was a req to not impact existing rules, but I wasn't clear on the mechanics of that (if we just needed to keep the ID the same). Should I switch it back?